packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -483,6 +488,10 @@ class ThemeData extends Diagnosticable {
   /// Defaults to the current platform.
   final TargetPlatform platform;
 
+  /// Whether to expand the tap target of certain material widgets to a minimum
+  /// of 48dp by 48dp.
+  final bool expandedTapTargetSize;
```

Comment:
This could be an enum or a double, but I don't think it makes sense for it to be a bool.